src/common/playlists/PlayList.jsx

```diff
@@ -5,6 +5,15 @@ import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
+  if(plays.length===0){
+    return(
+      <>
```

Comment:
You do not need the fragments, so remove them.. the top level div that you added will be enough